lib/auth/auth_with_roles.go

```diff
@@ -5400,7 +5415,8 @@ func (a *ServerWithRoles) CreateAppSession(ctx context.Context, req *proto.Creat
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	return session, nil
+
+	return session.WithoutSecrets(), nil
```

Comment:
I'm somewhat surprised this doesn't break anything.

Reply:
Yeah, the session secrets weren't used anywhere after `CreateAppSession`.

Reply:
`Snowflake` and `SamlIdP` sessions won't be affected by this change as they have their own unique endpoints. We would need to repeat this change to support MFA and Hardware Key support for those. From a glance, it looks like those don't use the secrets either so it should be simpler.